mysql-compatibility.md

```diff
@@ -10,6 +10,10 @@ TiDB is fully compatible with the MySQL 5.7 protocol and the common features and
 
 However, some features of MySQL are not supported. This could be because there is now a better way to solve the problem (such as XML functions superseded by JSON), or a lack of current demand versus effort required (such as stored procedures and functions). Some features might also be difficult to implement as a distributed system.
 
+- In addition, TiDB does not support the MySQL replication protocol, but provides specific tools to replicate data with MySQL.
+    - Replicate data from MySQL: [TiDB Data Migration (DM)](https://docs.pingcap.com/tidb-data-migration/stable/overview) is a tool for migrating data from MySQL/MariaDB into TiDB and replicating the incremental data.
```

Comment:
```suggestion
    - Replicate data from MySQL: [TiDB Data Migration (DM)](https://docs.pingcap.com/tidb-data-migration/stable/overview) is a tool for migrating data from MySQL/MariaDB into TiDB and supports replicating the incremental data.
```

seems strange if `migrating data` and `replicating the incremental data` are in same level